xml/release-notes.xml

```diff
@@ -334,6 +334,27 @@
     <package>libzypp</package> versions older than 12.25.8:
    </para>
    <screen>zypper dup --allow-vendor-change --force-resolution</screen>
+
+   <para>
+    &opensuseleap; 15.3 supplies all required rpm veryfication keys including the &sles;
+    as part of <package>openSUSE-build-key</package>.
+    All keys are also newly available in the oss repository.
+
+    <package>libzypp-17.25.11</package> imports all keys automatically.
```

Comment:
all -> required keys (keys will be checked and we will follow only keys with full fingerprint listed in the repofile) and if they're verified as trusted keys, then they'll be imported automatically and user will see that keys imported.

Reply:
Fixed